config/settings.yml

```diff
@@ -97,6 +97,7 @@
     :purge_window_size: 10000
 :ems:
   :ems_redhat:
+    :resolve_ip_addresses: true
```

Comment:
@jhernand - Thank you for providing more details.

The doc text you have provided in the bug is very clear and easy to follow. Given that users may be affected by this across versions, my feeling is that perhaps we should add this to the Managing Providers guide to help explain the behavior and offer a more long-term method for planning a deployment.